index.bs

```diff
@@ -189,6 +190,36 @@ longer publicly available due to any other means.
 
 ## Security and Privacy Considerations
 
+### High Risk of Pervasive Tracking
+
+The reliance of this DID Method on traditional HTTP-based Web Architecture can
+enable the same sorts of privacy-invasive tracking technologies that are in
+widespread use today (2020). For example, verification of a Verifiable
+Presentation might result in resolving a did:web DID by the verifier. This
+enables the DID Method registry to know when and where every did:web DID on the
+ledger is used, which is useful when tracking behaviour of the DID subject.
```

Comment:
What is "DID method registry" in this context? Does that mean the server which hosts the DID doc? I'd change the language to avoid confusion with [this DID method registry](https://www.w3.org/TR/did-spec-registries/#did-methods). We could say "Verifiable Data Registry" to be consistent with the DID Core spec?

In this case I also wouldn't use the word "ledger" here.

(I haven't reviewed the rest of the spec to make sure language aligns with the latest in DID Core, so I'll do that too.)

Reply:
+1 to using the term VDR.